stage_sdk_release.sh

```diff
@@ -0,0 +1,16 @@
+if [ -z $1 ] || [ -z $2 ]; then
+    echo "Usage: ./$0 [current_sdk_version] [new_sdk_version]"
+    exit 1
+fi
+
+export LC_CTYPE=C LANG=C
+echo "Bumping version from $1 to $2"
+find . -name 'AWSAppSync.podspec' -print0 | xargs -0 sed -i '' -e "s/$1/$2/g"
+find . -path '*AWSAppSyncClient/*.plist' -print0 | xargs -0 sed -i '' -e "s/$1/$2/g"
+find . -path '*AWSAppSyncClient/AWSAppSyncHTTPNetworkTransport.swift' -print0 | xargs -0 sed -i '' -e "s/$1/$2/g"
```

Comment:
I found a reference to the version number in AWSAppSyncClient/MQTTSDK/AWSIoTMQTTClient.m
That should be addressed as well.

Reply:
The AWSIoTMQTTClient.m SDK version is for the updates in MQTT SDK and should be inline with aws-sdk-ios MQTT client. I will switch the version to 2.6.19 to be inline in another commit.